mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -71,17 +71,24 @@ class Completions(
       case _ :: (_: Import) :: _ => false
       case (_: Ident) :: (_: SeqLiteral) :: _ => false
       case _ => true
-
   enum CursorPos:
     case Type(hasTypeParams: Boolean, hasNewKw: Boolean)
     case Term
     case Import
 
     def include(sym: Symbol)(using Context): Boolean =
+      def hasSyntheticCursorSuffix: Boolean =
```

Comment:
I changed it to version when we check if this completion is coming from a symbol we are currently creating